README.md

```diff
@@ -84,6 +84,53 @@ const subscription = pool.closeEpoch().subscribe(
 )
 ```
 
+## Investments
+
+Investments for a pool are done via vaults. Vaults can be deployed for a tranche on any supported chain for any supported currency.
+
+Retrieve a vault by querying it from the Pool:
+
+```js
+const pool = await centrifuge.pool('1')
+const vault = await pool.vault(1, '0xabc...', '0xdef...') // Chain ID, tranche ID, investment currency address
+```
+
+Query the state of an investment on the vault for an investor:
+
+```js
+const investment = await vault.investment('0x123...')
+// Will return an object containing:
+// isAllowedToInvest - Whether an investor is allowed to invest in the tranche
+// investmentCurrency - The ERC20 token that is used to invest in the vault
+// investmentCurrencyBalance - The balance of the investor of the investment currency
+// investmentCurrencyAllowance - The allowance of the vault
+// shareCurrency - The ERC20 token that is issued to investors to account for their share in the tranche
+// shareBalance - The number of shares the investor has in the tranche
+// claimableInvestShares - The number of shares an investor can claim after their invest order has been processed (partially or not)
+// claimableInvestCurrencyEquivalent - The equivalent value of the claimable shares denominated in the invest currency
+// claimableRedeemCurrency - The amout of money an investor can claim after their redeem order has been processed (partially or not)
+// claimableRedeemSharesEquivalent - The amount of shares that have been redeemed for which the investor can claim money
+// pendingInvestCurrency - The amount of money that the investor wants to invest in the tranche that has not been processed yet
+// pendingRedeemShares - The amount of shares that the investor wants to redeem from the tranche that has not been processed yet
+// claimableCancelInvestCurrency - The amount of money an investor can claim after an invest order cancellation has been processed
+// claimableCancelRedeemShares - The amount of shares an investor can claim after a redeem order cancellation has been processed
+// hasPendingCancelInvestRequest - Whether the investor has an invest order that is in the process of being cancelled
+// hasPendingCancelRedeemRequest - Whether the investor has a redeem order that is in the process of being cancelled
+```
+
+Invest in a vault:
+
+```js
+const result = await vault.increaseInvestOrder(1000)
```

Comment:
It currently accepts numbers, bigints and CurrencyBalances. If a number is passed, it does `number*10**decimals` in the function. My thinking was that it would be good to not force users to work with the decimals and do conversions. But maybe it's confusing that `increaseInvestOrder(1000)` and `increaseInvestOrder(1_000_000_000n)` will invest the same amount.

Reply:
Hm, I understand the perspective. I like the focus on simplifying user input, but do worry here that it will rather create issues with incorrect inputs.

I would actually lean towards only allowing `CurrencyBalance` as an input here. To force the user to clearly specify what the input denomination should be.

Reply:
It's not ideal, but maybe that's the way to go if we want to avoid confusion.